v22.2/ui-statements-page.md

```diff
@@ -7,18 +7,65 @@ docs_area: reference.db_console
 
 {% include {{ page.version.version }}/ui/admin-access.md %}
 
-The **Statements** page  of the DB Console helps you:
+The **Statements** page offers two views on statements: **Statement Fingerprints**, which represents one or more completed SQL statements and **Active Executions**,
+which represents individual statement executions in progress.
+
+You choose a view by selecting the **Statement Fingerprints** or **Active Executions** radio button. The selection is retained when you switch between the Statements and Transactions tabs on the SQL Activity page.
+
+{{site.data.alerts.callout_success}}
+If you haven't yet executed any queries in the cluster as a user, this page will be blank.
+{{site.data.alerts.end}}
+
+## Statement Fingerprints view
+
+The **Statements Fingerprints** view helps you:
 
 - Identify frequently executed or high latency [SQL statements](sql-statements.html).
 - View SQL statement fingerprint [details](#statement-fingerprint-page).
 - Download SQL statement [diagnostics](#diagnostics) for troubleshooting.
 
-To view this page, click **SQL Activity** in the left-hand navigation of the DB Console. The **Statements** tab is selected.
+{% if page.cloud != true %}
+To view this page, click **SQL Activity** in the left-hand navigation of the DB Console.
+{% else %}
+To view this page, click **SQL Activity** in the left-hand navigation of the {{ site.data.products.db }} Console.
+{% endif %}
 
-{{site.data.alerts.callout_success}}
-If you haven't yet executed any queries in the cluster as a user, this page will be blank.
+The **Statements** tab is selected. The **Statement Fingerprints** radio button is selected and the [Statements table](#statements-table) displays.
+
+The following screenshot shows the statement fingerprint for `SELECT city, id FROM vehicles WHERE city = $1` while running the [`movr` workload](cockroach-workload.html#run-the-movr-workload).
+
+<img src="{{ 'images/v22.2/statement-fingerprint.png' | relative_url }}" alt="Statement fingerprint" style="border:1px solid #eee;max-width:100%" />
+
+If you click the statement fingerprint in the **Statements** column, the [**Statement Fingerprint** page](#statement-fingerprint-page) displays.
+
+<img src="{{ 'images/v22.2/statement-details.png' | relative_url }}" alt="Statement details" style="border:1px solid #eee;max-width:100%" />
+
+## Active Executions view
+
+The **Active Executions** view helps you:
+
+- Understand and tune workload performance, particularly long-running statements.
+
+{% if page.cloud != true %}
+To display this view, click **SQL Activity** in the left-hand navigation of the DB Console.
+{% else %}
+To display this view, click **SQL Activity** in the left-hand navigation of the {{ site.data.products.db }} Console.
+{% endif %}
+
+The **Statements** tab is selected. Click the **Active Executions** radio button. The [Active Executions table](#active-executions-table) displays.
```

Comment:
```suggestion
When the **Statements** tab is selected, click the **Active Executions** radio button. The [Active Executions table](#active-executions-table) displays.
```

Reply:
When you click SQL Activity, the Statements tab and Statement Fingerprints button are selected by default.